### ISSUE TITLE
Adding glow

### ISSUE THREAD

**Issue by SJuras** [SJuras/ThrivingSource#1]
Any element that has border-box added to it in light mode 
( box-shadow: 0 5px 15px rgba(0,0,0,0.2); )
those are mostly box components and className="row-2" row class. 
should have a glow and a border in night-mode:

box-shadow: 0 5px 15px rgba(63, 52, 133, 1);
border: 2px solid rgba(63, 52, 133, 1);

However, 
things get a bit more complicated on "Services" page. 
We have 5 different sets of container components: 
ServicesPresentation1 which includes ServiceBox1 components,
ServicesPresentation2 which includes ServiceBox2 components,
ServicesPresentation3 which includes ServiceBox3 components,
ServicesPresentation4 which includes ServiceBox4 components,
and 
ServicesPresentation5 which includes ServiceBox5 components,

The main difference between those is the color of a ::after element in light-mode and color of glow and border in night-mode.
Basically, the color of ::after (top-right trangle in each box) is the same color as color of glow and border in nightmode.

component "ServiceBox1" in night-mode has these css rules added: 
box-shadow: 0 5px 15px rgba(63, 52, 133, 1);
border: 2px solid rgba(63, 52, 133, 1);

ServiceBox2 in night-mode should have: 
box-shadow: 0 5px 15px rgba(107, 56, 130, 1);
border: 2px solid rgba(107, 56, 130, 1);

ServiceBox3 in nightmode should have:
box-shadow: 0 5px 15px  rgba(160, 61, 126, 1);
border: 2px solid rgba(160, 61, 126, 1);

ServiceBox4: 
box-shadow: 0 5px 15px  rgba(221, 67, 123, 1);
border: 2px solid rgba(221, 67, 123, 1);

ServiceBox5:
box-shadow: 0 5px 15px  rgba(229, 68, 122, 1);
border: 2px solid rgba(229, 68, 122, 1);
